libp2p/crypto/crypto.nim

```diff
@@ -743,23 +799,23 @@ proc decodeExchange*(message: seq[byte],
 
 proc write*(vb: var VBuffer, pubkey: PublicKey) {.inline.} =
   ## Write PublicKey value ``pubkey`` to buffer ``vb``.
-  vb.writeSeq(pubkey.getBytes())
+  vb.writeSeq(pubkey.getBytes().get())
```

Comment:
hm, this seems a bit weak - `getBytes` can fail on invalid key, which would crash / Defect - is it reasonable to expect that the caller remembers to use valid keys here?

Reply:
I would indeed `Defect` due to the fact those helpers are used for internal stuff and every usage we have is kinda strict. 
Could go the `tryGet` route tho if you think it's better.